src/bundlers/node/nodeJsBinaryDependenciesBundler.ts

```diff
@@ -1,17 +1,27 @@
 import path from "path";
 import fs from "fs";
 import { BundlerInput, BundlerInterface, BundlerOutput, Dependency } from "../bundler.interface.js";
+import { CloudProviderIdentifier } from "../../models/cloudProviderIdentifier.js";
 import { fileExists } from "../../utils/file.js";
 import { log } from "../../utils/logging.js";
 import { debugLogger } from "../../utils/logging.js";
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import exec from "await-exec";
 import packageManager from "../../packageManagers/packageManager.js";
 import { UserError } from "../../errors.js";
+import { $ } from "execa";
+
+enum Architecture {
+    ARM = "arm64",
```

Comment:
Q: Why not ARM64 and AMD64?

Reply:
I feel like x86 is a more common name for the type of architecture that we are referring to.

Reply:
But both `arm` and `x86` are different architectures from `arm64` and `amd64`. You can compile for those as well using node-gyp

Reply:
Done.